PULL_REQUESTS.md

```diff
@@ -0,0 +1,203 @@
+-   Type of change in the PR title (bug fix, feature, breaking change)
+-   Checklist of items completed
+-   Screenshots/videos for visual changes
+-   Testing information
+
+Incomplete templates will delay the review process.
+
+## Labels and Their Meanings
+
+-   `ready-for-review`: PR is ready for maintainer review
+-   `work-in-progress`: PR is still being worked on, not ready for review
+-   `needs-changes`: PR requires changes based on review feedback
+-   `needs-tests`: PR is missing necessary tests
+-   `needs-docs`: PR requires documentation updates
+-   `needs-self-review`: PR requires that the author does a self-review of code to answer preliminary questions
+-   `needs-style-review`: PR needs to be checked by CSS and/or Design
+-   `breaking-change`: PR contains changes that break backward compatibility
+-   `help-wanted`: Extra attention is needed on this PR
+-   `on-hold`: PR is blocked and/or needs more discussion.
+-   `Spectrum CSS`: Contains a version bump of Spectrum CSS and will require a review by CSS expert
+-   `Component: [Name]`: PR effects this component
+
+Apply labels promptly to help maintainers prioritize and manage the review queue.
+
+## Pull Request Review Process
+
+### Review Timing
+
+-   Maintainers aim to review PRs within 2 business days
+-   If your PR hasn't received attention after 3 business days, feel free to ping the team in the PR comments
+
+### Review Expectations
+
+Reviewers will check for:
+
+1. Adherence to code style and component patterns
+2. Proper test coverage
+3. Documentation completeness
+4. Accessibility compliance
+5. Visual regression test coverage
+6. Performance considerations
+
+### Review Etiquette
+
+The code review process should be constructive, respectful, and focused on improvement. Both reviewers and PR authors should follow these guidelines:
+
+#### For Reviewers
+
+-   **Be Specific and Constructive**: Provide clear, actionable feedback rather than vague criticisms. Explain why a change is needed.
+-   **Prioritize Issues**: Focus on important issues first (architecture, functionality, performance) before minor stylistic concerns.
+-   **Ask Questions**: When something isn't clear, ask questions rather than making assumptions about intent.
+-   **Suggest Alternatives**: When pointing out issues, suggest possible solutions or approaches. Leverage the code suggestions feature to streamline applying feedback for the author.
+-   **Recognize Good Work**: Acknowledge well-written code and good design decisions. Positive reinforcement is valuable.
+-   **Remember Context**: Consider the PR author's experience level and the scope of changes when providing feedback.
+-   **Be Timely**: Complete reviews promptly to avoid blocking progress.
+-   **Avoid Nitpicking**: Focus on meaningful improvements rather than personal style preferences that don't violate project standards. Prepend comments with `nit:` to denote its non-blocking feedback.
```

Comment:
How can we reword this to focus on the positive outcome we're wanting?

Reply:
```suggestion
-   **Avoid Nitpicking**: Focus on meaningful improvements rather than personal style preferences that don't violate project standards. Prepend comments with `nit:` to denote it's non-blocking feedback.
```

nit: apostrophe 😉

Reply:
```suggestion
-   **Avoid Nitpicking**: Focus on meaningful improvements rather than personal style preferences that don't violate project standards. Prepend comments with `nit:` to denote that it is non-blocking feedback.
```